app.ts

```diff
@@ -0,0 +1,106 @@
+﻿/// <reference path="jquery.d.ts" />
+
+/**
+ * Michael Leibbrandt 
+ * Typescript Onboard Project
+ */
+
+namespace onboard {
+
+	// Variable declarations
+
+	let totRec = getMax(); // Total amount of records
+	let totHeadings;
+	let fromRecord = 0;
+	let toRecord = 15;
+	let count = 0;
+	let increase = 15;
```

Comment:
For these variables, define a type, e.g. `const totRec: number = getMax();`.  You can use const for the values that would not change.